app/views/users/edit.html.erb

```diff
@@ -23,6 +23,14 @@
               <%= form.text_field :phone_number, class: "form-control" %>
             </div>
 
+            <div class="input-style-1">
+              <%= form.label :date_of_birth, "Date of birth" %>
+              <%= form.text_field :date_of_birth,
+                                  value: @user.date_of_birth&.strftime("%Y/%m/%d"),
```

Comment:
Less concerned about this `date_of_birth&.strftime...` being in the decorator, but I'm not a huge fan of the safe operator in views.

Reply:
The decorator is a good place to handle nil DOB more explicitly anyways 👍